starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/Shopping_List.java

```diff
@@ -0,0 +1,61 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+/**
+ * Created by ds on 10/25/16.
+ */
+
+public class Shopping_List {
+    private int mID;
+    private String mItem_name;
```

Comment:
Best practice is to use "camel case" for your variable names, and not include any underscores. For example, this should be "mItemName" rather than "mItem_name"